config/locales/en.yml

```diff
@@ -1410,7 +1410,7 @@ en:
       members: Members
       samples: Samples
       settings: Settings
-      workflow_executions: Workflow Executions
+      workflow_executions: Pipeline Executions
```

Comment:
**nitpick (blocking):** Can you revert the following 5 translations.

Reply:
Updated in 51f582d